src/meili/map_matcher.cc

```diff
@@ -277,31 +277,43 @@ MapMatcher::OfflineMatch(
     const std::vector<Measurement>& measurements, uint32_t k) {
   mapmatching_.Clear();
 
-  const auto begin = measurements.begin(),
-               end = measurements.end();
-
-  if (begin == end) {
+  if (measurements.size() < 2)
```

Comment:
Why do we need at least 2 measurements while a single measurement can match the closest road segment?